Automation_Test_Framework/TestSetup/NUnitWebTestSetUp.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Core_Framework.NUnitTestSetup;
+using NUnit.Framework;
+
+namespace Automation_Test_Framework.TestSetup;
+
+public class NUnitWebTestSetUp : NUnit_Test_Setup
```

Comment:
NUnitWebTestSetUp NUnitAPITestSetUp NUnitWebAndAPITestSetUp 
They have the same purpose. So please don't divide the base test into many parts

Reply:
Done!